source/community/reactnative/.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Is this file required to be in GitHub?

Reply:
Have created a sample react native project with the latest react-native 0.62.0.

It includes .prettierrc.js file. I can also see the same configuration file in the other latest projects too.